### ISSUE TITLE
New private message/group does not appear after making a new message group

### ISSUE THREAD

**Issue by swallitsch** [zulip/zulip-mobile#3460]
iOS 12.1 & 12.2
Zulip 23.2.111

Clicking "Create Group" allows me to compose a new private message for someone, but when I go back to the PM screen, that group or recipient does not show up in the list. The only way to get to it is from the "All Messages" screen, or by force quitting and relaunching the app.

This bug has a decently high impact on our workflows

**Comment by swallitsch** [comment]
Sounds like this might get automatically fixed when support for #3133 hits.

**Comment by m4tf3r** [comment]
After looking at server code, I fixed this by setting in constants.js:
`export const LAST_MESSAGE_ANCHOR = 1e16;`
From what I understood, for messages events to work as intended, the server must get exactly this number as message anchor (the number is set in zerver/views/messages.py, line 64).

**Comment by chrisbobbe** [comment]
Thanks for the report and investigation! As it happens, it looks like we’re very near to resolving this in https://github.com/zulip/zulip-mobile/issues/3654#issuecomment-580938373. Please let me know if I’ve closed this in error; pinging @gnprice to double-check.

**Comment by gnprice** [comment]
I just attempted to repro this at the current version from master (d7b68bddc, coming after #3654 was merged), and it indeed appears to be fixed!

One key element of this bug description is that this is about a case where you've actually *sent* a message to that new group of people. If you do "Create Group", and look at the message-list screen for that group, but don't actually send (or receive) a message there, then when you go back to the PM conversations screen you still won't see the group there. That other behavior isn't exactly a bug -- it's a natural consequence of the Zulip way of thinking about PM conversations, and the webapp has the exact same behavior, though it might well be surprising.

In any case that behavior is pretty independent of this issue, so if we wanted to discuss changing it that should be a new issue. Mentioning it here in case someone comes looking for an issue about that behavior and finds this one :slightly_smiling_face:
